cypress/e2e/EngageSphere_API.cy.js

```diff
@@ -0,0 +1,38 @@
+const CUSTOMERS_API_URL= `${Cypress.env('API_URL')}/customers`;
+
+
+describe('GET /customers', () => {
+  it('Retorna o código de status 200 em uma requisição bem sucedida', () => {
+      cy.request('GET', 'http://localhost:3001/customers?page=2&limit=10&size=All')
+      .then(({ status }) => {
+        expect(status).to.equal(200);
+        });
```

Comment:
Atenção com a indentação.

```suggestion
      });
```